multicall-0xsequience/index.html

```diff
@@ -22,6 +22,9 @@
       <p>RIF balance: <span id="rif-balance"></span></p>
```

Comment:
You should rename the directory, there's  a small typo: `multicall-0xsequience/` --> `multicall-0xsequence/`